dashboard-ui/src/components/App/index.jsx

```diff
@@ -150,16 +144,15 @@ export class App extends React.Component {
                                 <li className="nav-item">
                                     <Link className="nav-link" to="/">Home</Link>
                                 </li>
-                                <li className="nav-item">
-                                    <Link className="nav-link" to="/results">Results</Link>
-                                </li>
-                                <li className="nav-item">
-                                    <Link className="nav-link" to="/scenarios">Scenarios</Link>
-                                </li>
-                                <li className="nav-item">
-                                    <Link className="nav-link" to="/actionsCompare">Actions Compare</Link>
-                                </li>
-                                <NavDropdown title="Surveys">
+                                <NavDropdown title="ADM Results">
+                                    <NavDropdown.Item>
+                                        <Link className="dropdown-item" to="/results">Results by ADM</Link>
+                                    </NavDropdown.Item>
+                                    <NavDropdown.Item>
+                                        <Link className="dropdown-item" to="/scenarios">ADM Scenarios</Link>
+                                    </NavDropdown.Item>
+                                </NavDropdown>
+                                <NavDropdown title="Delegation Surveys">
```

Comment:
Forgot to mention this in my review. This causes errors in the console every time you click on a nav. An easy way to remedy this is add as={Link}, the `to` field and the `className` field to NavDropdown.Item, then just remove link completely. Examples should be below this piece of code where I've made this fix